src/main/java/de/infynyty/zuap/insertionHandler/WGZimmerHandler.java

```diff
@@ -26,27 +29,52 @@ public WGZimmerHandler(@NotNull String logPrefix, @NotNull InsertionAnnouncer an
 
     //TODO: Make it possible to change search variables
     @Override
-    protected String pullUpdatedData() throws IOException, InterruptedException {
-        HttpRequest wgZimmerRequest = HttpRequest.newBuilder()
-            .uri(URI.create("https://www.wgzimmer.ch/wgzimmer/search/mate.html?"))
-            .header("Accept", "text/html,application/xhtml+xml,application/xml;q=0.9,image/avif,image/webp,*/*;q=0.8")
-            .header("Accept-Encoding", "gzip, deflate, br")
-            .header("Content-Type", "application/x-www-form-urlencoded")
-            .POST(HttpRequest.BodyPublishers.ofString("query=&priceMin=200&priceMax=650&state=all&permanent"
-                + "=all&student=true&typeofwg=all&orderBy=%40sortDate&orderDir=descending&startSearchMate=true&wgStartSearch=true&start=0"))
-            .build();
+    protected String pullUpdatedData() {
+        var options = new FirefoxOptions();
```

Comment:
For consistency reasons, please avoid the usage of `var`.

Reply:
done